tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/expressions/stamp/StampInbuiltFunctionNegativeTest.java

```diff
@@ -86,7 +86,7 @@ public void testStampNegativeTest() {
                 "incompatible types: expected 'typedesc<anydata>', found 'typedesc<EmployeeObject>'",
                 87, 63);
         BAssertUtil.validateError(compileResult, index++,
-                "incompatible types: expected 'typedesc<anydata>', found 'typedesc<map>'", 95, 54);
+                "incompatible types: expected 'typedesc<anydata>', found 'typedesc<AnyMap>'", 95, 54);
```

Comment:
Please add a test or two where the referred type is another reference type.

E.g.,

```ballerina
type AnyMap map<any>;
type MapAny AnyMap;

public function main() {
    anydata[] anyArray = [];
    map<any>|error x = anyArray.cloneWithType(MapAny);
    map<any>|error y = anyArray.cloneWithType(AnyMap);
}
```

Reply:
Added